smrf/tests/data/test_hrrr_grip.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+
+import pandas as pd
+
+import smrf.data as smrf_data
+
+
+class TestInputGribHRRR(unittest.TestCase):
```

Comment:
File name is `grip`, did you mean `grib`?

Reply:
Good catch!